docs/docs/index.md

```diff
@@ -65,6 +46,21 @@ See [**official Terraform documentation**](https://developer.hashicorp.com/terra
 
         2. Move it to a directory under `PATH`, or add the directory where you'd like to keep the executable to `PATH`.
 
+    === "Build From Source"
+
+        ```shell
+        # Clone the repo
+        $ git clone git@github.com:schrodinger/infra-tester.git
+
+        # Build the executable and move the binary to a directory in the $PATH
+        $ cd infra-tester && go build -o bin/infra-tester
+        $ sudo mv infra-tester /usr/local/bin
```

Comment:
This step seems Linux-specific.

Reply:
It's the same in Mac as well. `/usr/local/bin` contains a lot of user installed binaries.